models/list_options.go

```diff
@@ -41,7 +41,7 @@ func (opts *ListOptions) setEnginePagination(e Engine) Engine {
 func (opts *ListOptions) GetStartEnd() (start, end int) {
 	opts.setDefaultValues()
 	start = (opts.Page - 1) * opts.PageSize
-	end = start + opts.Page
+	end = start + opts.PageSize
```

Comment:
I think the old implementation had an issue as the end should be start + pageSize and not 
start + Page 
or did i miss anything here?

Reply:
agh!

This needs to get into 1.15 and backport to 1.14 can you break this out to another PR?

Reply:
sure.

Reply:
https://github.com/go-gitea/gitea/pull/16303